tests/integ/test_dataframe.py

```diff
@@ -898,6 +903,9 @@ def test_filter(session):
     expected = []
     assert res == expected
 
+
+def test_filter_with_sql_str(session):
```

Comment:
I separated the part from the original `test_filter` that specifies condition as `str` into a this new test.

Reply:
out of curiosity, a string condition will be passed into the execution plan as it is?

I'm maintaining a list of unsupported cases so thanks for catching this